libs/shared/assistant/feature/src/lib/hit/hit.tsx

```diff
@@ -0,0 +1,47 @@
+import { type PropsWithChildren } from 'react'
+import Snippet from '../snippet/snippet'
+
+function Wrapper({ children }: PropsWithChildren) {
+  return <span className="flex flex-col gap-0.5">{children}</span>
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function Hit({ hit: item }: any) {
```

Comment:
I remember it was inspired by an existing component from Algolia, but I don't find it. If you ever remember it would be cool to comment above on the link

Reply:
Done https://github.com/Qovery/console/pull/1278/commits/6cb70c6e66da01f74b4d924f4101440091123972